client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/site-setup-design-picker.tsx

```diff
@@ -165,6 +167,11 @@ const SiteSetupDesignPicker: Step = ( { navigation, flow } ) => {
 				generated_designs: generatedDesigns?.map( ( design ) => design.slug ).join( ',' ),
 			} );
 		}
+
+		setStepData( {
```

Comment:
We have to create another `useEffect` for it as the value replies on `isMobile`. However, do we need to check `isMobile` as we only use the `position: fixed;` on the desktop viewport?

Reply:
You are right, we can use css only